plugins/ipam/host-local/backend/allocator/range.go

```diff
@@ -49,7 +49,7 @@ func (r *Range) Canonicalize() error {
 	// If the gateway is nil, claim .1
 	if r.Gateway == nil {
 		r.Gateway = ip.NextIP(r.Subnet.IP)
-	} else {
+	} else if r.Gateway.To4() != nil {
```

Comment:
How about to add canonicalizeIP() for IPv6? It seems to just skip code in case of IPv6.

Reply:
Ok, added check after Canonicalize.